msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/AccountState.kt

```diff
@@ -207,13 +210,11 @@ class AccountState private constructor(
     interface GetAccessTokenCallback : Callback<GetAccessTokenResult>
 
     /**
-     * Retrieves the access token for the currently signed in account from the cache.
+     * Retrieves the access token for the default OIDC(openid, offline_access, profile)  scopes from the cache
```

Comment:
nit, whitespace missing between C and (

Reply:
and nit: double whitespace before "scopes"

Reply:
Done